composer.json

```diff
@@ -11,7 +11,8 @@
   "require": {
     "php": "^7.4 || ^8",
     "ext-json": "*",
-    "ext-mbstring": "*"
+    "ext-mbstring": "*",
+    "psr/simple-cache": "^1.0"
```

Comment:
I would like to keep this library dependency-free. Let's define our own interface or implement this in terms of a callable instead.

Reply:
Sure, no prob. Will update.

Reply:
Updated.